v22.1/aws-dms.md

```diff
@@ -24,6 +24,15 @@ Complete the following items before starting this tutorial:
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.Creating.html) in AWS.
 - Configure a [source endpoint](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html) in AWS pointing to your source database.
 - Ensure you have a secure, publicly available CockroachDB cluster running v22.1.8 or later.
+    - If your CockroachDB cluster is running v22.1.14 or later, set the following session variable using [`ALTER ROLE ... SET {session variable}`](alter-role.html#set-default-session-variable-values-for-a-role):
```

Comment:
Maybe this goes without saying, but I'm wondering if we should say here that if you're migrating large tables (and define what large is), that customers should upgrade to at least 22.1.14

Reply:
@ajstorm I've added this, but since we recommend v22.2.4 by default on 22.2 clusters, would it be simpler to recommend v22.1.14 by default on 22.1 (or is that going too far)?

Reply:
22.1.14 will have the requisite changes